en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
 # Monitoring Server Health
 
-!!! warning
-This document is work in progress!
+**Health Check API** is used to validate and verify the status of the APIM server and its dependencies
+It is a REST service which returns the operational status, performance information of the configured endpoints with appropriate HTTP return code and a JSON object
+There are two health checkers available by default:
 
-
-The Carbon Health Check API can be used to check the health of a Carbon server. The sections below guide you through using this API.
-
--   [Deploying the API](#MonitoringServerHealth-DeployingtheAPI)
--   [Configuring the API](#MonitoringServerHealth-ConfiguringtheAPI)
--   [Invoking the API](#MonitoringServerHealth-InvokingtheAPI)
--   [Error responses](#MonitoringServerHealth-Errorresponses)
--   [Adding new health checkers](#MonitoringServerHealth-Addingnewhealthcheckers)
+1.  **Data sources health checker** - This checker goes through the data sources that are configured in the `<API-M_HOME>/repository/conf/deployment.toml` file and checks if the active connection count surpasses a healthy percentage limit (e.g., 80%) of the maximum allowed connections count. This checker also tests the connection from each data source to see whether the connection is successful.
```

Comment:
can we configure the healthy percentage limit?
The "connection is successful"  means whether it is alive connection?

Reply:
Yes we can configure it using `pool_usage_limit_percentage`  . This is documented in **DataSource Health Checker** section.

yes it is an alive connection